tests/contrib/django/test_connection.py

```diff
@@ -30,7 +26,7 @@ def test_connection(self):
 
         span = spans[0]
         eq_(span.name, 'sqlite.query')
-        eq_(span.service, 'defaultdb')
+        eq_(span.service, 'sample-defaultdb')
```

Comment:
@palazzem how can I run only django test suit? When I try `python tests/contrib/django/runtests.py` I get erro:

```bash
Traceback (most recent call last):
  File "tests/contrib/django/runtests.py", line 18, in <module>
    execute_from_command_line([sys.argv[0], "test", app_to_test])
  File "/Users/jsouza/.virtualenvs/ddtrace/lib/python3.6/site-packages/django/core/management/__init__.py", line 363, in execute_from_command_line
    utility.execute()
  File "/Users/jsouza/.virtualenvs/ddtrace/lib/python3.6/site-packages/django/core/management/__init__.py", line 355, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/jsouza/.virtualenvs/ddtrace/lib/python3.6/site-packages/django/core/management/commands/test.py", line 29, in run_from_argv
    super(Command, self).run_from_argv(argv)
  File "/Users/jsouza/.virtualenvs/ddtrace/lib/python3.6/site-packages/django/core/management/base.py", line 275, in run_from_argv
    parser = self.create_parser(argv[0], argv[1])
  File "/Users/jsouza/.virtualenvs/ddtrace/lib/python3.6/site-packages/django/core/management/base.py", line 249, in create_parser
    self.add_arguments(parser)
  File "/Users/jsouza/.virtualenvs/ddtrace/lib/python3.6/site-packages/django/core/management/commands/test.py", line 50, in add_arguments
    test_runner_class = get_runner(settings, self.test_runner)
  File "/Users/jsouza/.virtualenvs/ddtrace/lib/python3.6/site-packages/django/test/utils.py", line 317, in get_runner
    test_runner_class = settings.TEST_RUNNER
  File "/Users/jsouza/.virtualenvs/ddtrace/lib/python3.6/site-packages/django/conf/__init__.py", line 56, in __getattr__
    self._setup(name)
  File "/Users/jsouza/.virtualenvs/ddtrace/lib/python3.6/site-packages/django/conf/__init__.py", line 39, in _setup
    % (desc, ENVIRONMENT_VARIABLE))
django.core.exceptions.ImproperlyConfigured: Requested setting TEST_RUNNER, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.```

I write a test with `override_settings` but it doesn't work on ci an I don't know how to debug locally.